server/datastore/mysql/hosts.go

```diff
@@ -503,24 +503,29 @@ const hostMDMSelect = `,
 			WHEN hdek.decryptable IS NULL OR hdek.decryptable = 0 THEN CAST(FALSE AS JSON)
 			ELSE CAST(TRUE AS JSON)
 		END,
+		'raw_decryptable',
+		CASE
+			WHEN hdek.host_id IS NULL THEN -1
+			ELSE hdek.decryptable
+		END,
```

Comment:
I wonder if as a post-MVP cleanup task we should get rid of `encryption_key_available` here and use `raw_decryptable` upstream.